features/networking/sctp.feature

```diff
@@ -0,0 +1,59 @@
+Feature: SCTP related scenarios
+
+  # @author weliang@redhat.com
+  # @case_id OCP-28757
+  @admin
+  @destructive
+  Scenario: Establish pod to pod SCTP connections		
+    Given I store the workers in the :workers clipboard
+    And the Internal IP of node "<%= cb.workers[1].name %>" is stored in the :worker1_ip clipboard
+    
+    Given I install load-sctp-module
+    # Installing sctp module will reboot all the nodes and it will more than 10 minutes
+    And I wait up to 800 seconds for the steps to pass:
+    """
+    Given 60 seconds have passed 
+    When I run the :get admin command with:
+      |resource | nodes |
+    Then the outputs should not contain "SchedulingDisabled"
+    """
+    
+    Given I have a project
+    Given I obtain test data file "networking/sctp/sctpserver.yaml"
+    When I run oc create as admin over "sctpserver.yaml" replacing paths:
+      | ["metadata"]["namespace"] | <%= project.name %>       |
+      | ["spec"]["nodeName"]      | <%= cb.workers[0].name %> |
+    Then the step should succeed
+    And the pod named "sctpserver" becomes ready
+    Then evaluation of `pod.ip` is stored in the :serverpod_ip clipboard
+
+    Given I obtain test data file "networking/sctp/sctpclient.yaml"
+    When I run oc create as admin over "sctpclient.yaml" replacing paths:
+      | ["metadata"]["namespace"] | <%= project.name %>       |
+      | ["spec"]["nodeName"]      | <%= cb.workers[1].name %> |
+    Then the step should succeed
+    And the pod named "sctpclient" becomes ready
+   
+    # sctp server pod start to wait for sctp traffic
+    When I run the :exec background admin command with:
+      | pod              | sctpserver          |
+      | namespace        | <%= project.name %> |
+      | oc_opts_end      |                     |
+      | exec_command     | sh                  |
+      | exec_command_arg | -c                  |
+      | exec_command_arg | nc -l 30102 --sctp  |     
+    Then the step should succeed
+    Given 10 seconds have passed
+
+    # sctp client pod start to send traffic 
+    When I run the :exec background admin command with:
+      | pod              | sctpclient                                                         |
+      | namespace        | <%= project.name %>                                                |
+      | oc_opts_end      |                                                                    |
+      | exec_command     | sh                                                                 |
```

Comment:
Please revisit indentation.

Reply:
@anuragthehatter Could you please check if you still see the indentation between line 46 and 51 in new commit? Thanks!

Reply:
Looks good, Thanks Weibin!